sys/fuchsia/fidl.txt

```diff
@@ -25,7 +25,8 @@ type fidl_message_header[METHOD_ORDINAL] {
 	txid		const[0, int32]
 	reserved	const[0, int32]
 	flags		const[0, int32]
-	ordinal		const[METHOD_ORDINAL, int32]
+# TODO: this should be int32, but the consts passed here don't fit into 32 bits.
```

Comment:
@mvanotti There is something very wrong with all fidl descriptions. If I am reading this correctly, we always passed method ordinal 0 for all commands. So probably never really tested any of this...
I am going to submit this soon. Please submit a proper fix afterwards. Either consts need to be changed, or field type, or something else, I don't know what's the right fix.
Thanks